treeherder/config/settings.py

```diff
@@ -453,10 +473,17 @@
 BUGFILER_API_URL = env("BUGZILLA_API_URL", default=BZ_API_URL)
 BUGFILER_API_KEY = env("BUGZILLA_API_KEY", default=None)
 
+# For intermittents commenter
+COMMENTER_API_KEY = env("COMMENTER_API_KEY", default=None)
```

Comment:
The existing bug filter API key is set via the environment variable `BUGZILLA_API_KEY`. Now that we'll have two API keys, I think we might want to make that name more specific too. Also, I wonder if it's clearer (for someone looking at the Heroku environment variables, wondering what each of them is for), for the name to include "bugzilla"? ("Commenter" could be an external service etc).

So what do you think about something like:
`BUG_COMMENTER_API_KEY`
`BUG_FILER_API_KEY`

(open to alternatives)

Prior to merging this PR, we'll need to ensure that `BUG_FILER_API_KEY` is set in all the places where `BUGZILLA_API_KEY` is currently set (whilst leaving `BUGZILLA_API_KEY` in place for now). Plus in production **only** we'll want to set `BUG_COMMENTER_API_KEY`.

Reply:
Sure. Do you want me to change `BUGZILLA_API_KEY` in a separate commit or separate issue/pr?

Reply:
In this PR is fine, since it's related (in that there is only the need to differentiate the name, due to there now being two bugzilla API keys)